### PR TITLE
12.0 mig purchase batch invoicing

### PR DESCRIPTION
**Pull request by amcor**
cc @guadaltech

### CODE REVIEW

File: purchase_batch_invoicing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Batch Invoicing",
+    "summary": "Make invoices for all ready purchase orders",
+    "version": "12.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://www.github.com/OCA/account-invoicing",
```
Comment (nikul-serpentcs) [inline purchase_batch_invoicing/__manifest__.py]: `https://github.com/OCA/account-invoicing`

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: purchase_batch_invoicing/tests/test_invoicing.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            "user_type_id": cls.account_type1.id,
+            "reconcile": True,
+        })
+        cls.vendor1 = cls.env["res.partner"].create({
+            "name": "Vendor 1",
+            "supplier": True,
+            "is_company": True,
+            "property_account_payable_id": cls.account1.id,
+        })
+        cls.uomcateg1 = cls.env["uom.categ"].create({
```
Comment (oihane) [inline purchase_batch_invoicing/tests/test_invoicing.py]: @amcor model is `uom.category`

File: purchase_batch_invoicing/tests/test_invoicing.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            "name": "UOM 1",
+            "category_id": cls.uomcateg1.id,
+            "factor": 1,
+            "active": True,
+            "uom_type": "reference",
+        })
+        cls.product1 = cls.env["product.product"].create({
+            "name": "Product 1",
+            "purchase_ok": True,
+            "type": "product",
```
Comment (oihane) [inline purchase_batch_invoicing/tests/test_invoicing.py]: You can only define a product type as "Stockable" if `stock` module is installed, change it to `'consu'`

**Comment by pedrobaeza** [comment]
Superseded by #625 as this is red

**Pull request closed without merging**